dept44-starter-webclient/src/test/java/se/sundsvall/dept44/configuration/webclient/WebClientBuilderTest.java

```diff
@@ -2,69 +2,85 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
 import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.mockStatic;
 import static org.mockito.Mockito.when;
 
 import java.time.Duration;
+import java.util.Set;
 
 import org.assertj.core.api.InstanceOfAssertFactories;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentMatchers;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.HttpStatusCode;
+import org.springframework.http.ResponseEntity;
 import org.springframework.security.oauth2.client.registration.ClientRegistration;
 import org.springframework.security.oauth2.client.web.reactive.function.client.ServerOAuth2AuthorizedClientExchangeFilterFunction;
 import org.springframework.web.reactive.function.client.WebClient;
+import org.springframework.web.reactive.function.client.support.WebClientAdapter;
+import org.springframework.web.service.annotation.GetExchange;
+import org.springframework.web.service.invoker.HttpClientAdapter;
+import org.springframework.web.service.invoker.HttpServiceProxyFactory;
 import org.zalando.logbook.Logbook;
+import org.zalando.problem.Problem;
+import org.zalando.problem.Status;
 
 import io.netty.channel.ChannelOption;
-import se.sundsvall.dept44.support.BasicAuthentication;
+import reactor.core.publisher.Mono;
 
+@ExtendWith(MockitoExtension.class)
 class WebClientBuilderTest {
+
 	private static final String BASE_URL = "baseUrl";
 	private static final String USER_NAME = "userName";
 	private static final String PASSWORD = "password";
 	private static final Duration CONNECT_TIMEOUT = Duration.ofSeconds(54);
 	private static final Duration READ_TIMEOUT = Duration.ofSeconds(55);
 	private static final Duration WRITE_TIMEOUT = Duration.ofSeconds(56);
+
 	private static final Logbook LOGBOOK_MOCK = mock(Logbook.class);
 	private static final ClientRegistration CLIENT_REGISTRATION_MOCK = mock(ClientRegistration.class);
 
 	@Test
 	void testNullFields() {
```

Comment:
"customizers" is excluded from null checks. Should the field have a "hasFieldOrPropertyWithValue" check like the other excluded properties?

Reply:
Should we include the bumb of versions suggested by dependabot in this PR?

Reply:
Yes, definitely and yes, why not